ugs-core/src/com/willwinder/universalgcodesender/AbstractController.java

```diff
@@ -886,10 +888,6 @@ protected void dispatchConsoleMessage(MessageType type, String message) {
     }
     
     protected void dispatchStateChange(ControlState state) {
-        if (this.currentState == COMM_SENDING_PAUSED && state != this.currentState
-                && this.comm.isPaused() && !this.isStreaming()) {
-            this.comm.resumeSend();
-        }
```

Comment:
This block was only relevant when `dispatchStateChange` is called from `communicatorPausedOnError`, so I moved it there.